frontend/src/pages/ReviewWritingPage/form/hooks/question/useLoadAndPrepareReview.ts

```diff
@@ -1,16 +1,59 @@
+import { useMemo } from 'react';
+
+import { ReviewWritingCardSection } from '@/types';
+import substituteString from '@/utils/substituteString';
```

Comment:
```suggestion
import {substituteString}  from '@/utils';
```

utils/index에서 subtituteString을 export 한 후  위와 같이 사용하는 것은 어떨까요?

Reply:
util의 index에 누락되어 있던 함수를 추가했어요